src/containers/NoteArea/NoteArea.test.js

```diff
@@ -8,9 +8,9 @@ describe('NoteArea', () => {
 
   it('should match the snapshot', () => {
     //setup
-    const mockItems = [{ description: 'Note one of Title one', noteID: 1, id: 1 }, { description: 'Note two of Title one', noteID: 1, id: 2 }, { description: 'Note three of Title one', noteID: 1, id: 3 },
-    { description: 'Note one of Title two', noteID: 2, id: 1 }, { description: 'Note two of Title two', noteID: 2, id: 2 }]
-    const mockNotes = [{ id: 1, title: "Title One" }, { id: 2, title: "Title Two" }]
+    const mockItems = [{ description: 'Note one of Title one', noteID: '1', id: '1' }, { description: 'Note two of Title one', noteID: '1', id: '2' }, { description: 'Note three of Title one', noteID: '1', id: '3' },
+    { description: 'Note one of Title two', noteID: '2', id: '1' }, { description: 'Note two of Title two', noteID: '2', id: '2' }]
+    const mockNotes = [{ id: '1', title: "Title One" }, { id: '2', title: "Title Two" }]
```

Comment:
Looks like the similar issue Travis was working on in the earlier tests - glad we got it sorted out!